bin/lib/check_reqs.js

```diff
@@ -75,20 +78,74 @@ module.exports.help = function () {
  * @return {Promise}           Returns a promise either resolved with tool version or rejected
  */
 function checkTool (tool, minVersion, optMessage) {
-    if (os.platform() !== 'darwin'){
-        // Build iOS apps available for OSX platform only, so we reject on others platforms
-        return Q.reject('Cordova tooling for iOS requires Apple OS X');
-    }
     // Check whether tool command is available at all
     var tool_command = shell.which(tool);
     if (!tool_command) {
-        return Q.reject(optMessage || (tool + 'command is unavailable.'));
+        return Q.reject(tool + ' was not found. ' + (optMessage || ''));
```

Comment:
I presume the `opt` in `optMessage` stands for 'optional` - it's not longer really optional, so perhaps change the name?